Add K8S Network Section & Fix some typos (#142)
binbashar/le-ref-architecture-doc: docs/how-it-works/features/compute/k8s-eks/vpc-adressing.md

    In order to size a VPC correctly, it is important to understand the number, types, and sizes of workloads 
    expected to run in it, as well as workload elasticity and load balancing requirements. 
    
    Keep in mind that there is no charge for using Amazon VPC (aside from EC2 charges), therefore cost 
    should not be a factor when determining the appropriate size for your VPC, so make sure you size your 
    VPC for growth.
    
    Moving workloads or AWS resources between networks is not a trivial task, so be generous in your 
    IP address estimates to give yourself plenty of room to grow, deploy new workloads, or change your 
    VPC design configuration from one to another. The majority of AWS customers use VPCs with a /16 
    netmask and subnets with /24 netmasks. The primary reason AWS customers select smaller VPC and 
    subnet sizes is to avoid overlapping network addresses with existing networks. 

    So having [AWS single VPC Design](https://aws.amazon.com/answers/networking/aws-single-vpc-design/) we've chosen
    a Medium/Small VPC/Subnet addressing plan which would probably fit a broad range variety of
    use cases

## Networking - IP Addressing

> !!! example "Starting CIDR Segment (AWS Org)"

Reviewer: @marianod92 I think here we should update this for a reference to the "EKS clusters", suggesting the commit below:

```suggestion
!!! example "Starting CIDR Segment (AWS EKS clusters)"
```

Reviewer: Thanks @marianod92!

>     * [x] AWS Org IP Addressing calculation is presented below based on segment `10.0.0.0/16` (starts at /16 due to AWS VPC limits)

Reviewer: @marianod92 here we should also refer to the "EKS clusters". Something similar to the suggestion below should work:

```suggestion
    * [x] AWS EKS clusters IP Addressing calculation is presented below based on segment `10.0.0.0/16` (starts at /16 due to AWS VPC limits)
```

Author: Done!

Reviewer: Thanks! 🚀

    * [x] We started from `10.0.0.0/16` and subnetted to `/19`
    * [x] Resulting in **Total Subnets: 8**
      *   Number of available hosts for each subnet: 8190
      *   Number of available IPs (AWS) for each subnet: 8187
    
!!! example "Individual CIDR Segments (VPCs)"
    :fast_forward: Then each of these are /16 to /19
    

>     *   [x] Considering the whole Starting CIDR Segment (AWS Org) before declared, we'll start at `10.0.0.0/16`

Reviewer: @marianod92 same than before comments referring to the clusters, possible suggestion below:

```suggestion
    *   [x] Considering the whole Starting CIDR Segment (AWS EKS clusters) before declared, we'll start at `10.0.0.0/16`
```

Author: Done!

Reviewer: Great!
